ogorek.go

```diff
@@ -86,6 +86,7 @@ const (
 var errNotImplemented = errors.New("unimplemented opcode")
 var ErrInvalidPickleVersion = errors.New("invalid pickle version")
 var errNoMarker = errors.New("no marker in stack")
+var errStackOverflow = errors.New("pickle: stack overflow")
```

Comment:
I'm not sure about the wording of this, isn't it technically a stack underflow rather than an overflow?